ses

```diff
@@ -46,133 +46,156 @@ section_header "Ceph cluster status"
-        while read image; do
-            plugin_command "timeout $CT rbd ${ID} -p $pool info $image" \
-                > $LOG/ceph/images/${pool}/rbd-info-${image} 2>&1
+        # timeout status commands after 5 seconds
+        CT=5
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT versions" > $LOG/ceph/ceph-versions 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT -s" > $LOG/ceph/ceph-status 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT health detail" > $LOG/ceph/ceph-health-detail 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT config dump" > $LOG/ceph/ceph-config-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT mon dump" > $LOG/ceph/ceph-mon-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT mgr dump" > $LOG/ceph/ceph-mgr-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT osd tree" > $LOG/ceph/ceph-osd-tree 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT osd df tree" > $LOG/ceph/ceph-osd-df-tree 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT osd dump" > $LOG/ceph/ceph-osd-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT osd pool autoscale-status" > $LOG/ceph/ceph-osd-pool-autoscale-status 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT df detail" > $LOG/ceph/ceph-df-detail 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT fs dump -f json-pretty" > $LOG/ceph/ceph-fs-dump.json 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT fs dump" > $LOG/ceph/ceph-fs-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT pg dump -f json-pretty" > $LOG/ceph/ceph-pg-dump.json 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT pg dump" > $LOG/ceph/ceph-pg-dump 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT auth list" 2>&1 |
+            grep -v 'installed auth entries' |
+            sed "s/\(key:\) .*/\1 $CENSORED/g" > $LOG/ceph/ceph-auth-list
+        # `ceph report` does actually include the above information, but
+        # in JSON format.  Since adding `ceph report`, the above commands
+        # remain, because their output is easier to read in a hurry ;)
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT report" > $LOG/ceph/ceph-report 2>&1
+        plugin_command "${CEPHADM_SHELL} timeout $CT rados ${ID} df" > $LOG/ceph/rados-df 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT telemetry status" > $LOG/ceph/ceph-telemetry-status 2>&1
+        plugin_command "${CEPHADM_SHELL} ceph ${ID} --connect-timeout=$CT balancer status" > $LOG/ceph/ceph-balancer-status 2>&1
+
```

Comment:
```bash
ceph orch status >> $LOG/ceph/cephadm-status 
ceph cephadm get-ssh-config >> $LOG/ceph/cephadm-status 
ceph orch ls --format yaml >> $LOG/ceph/cephadm-status 
ceph orch host ls --format yaml >> $LOG/ceph/cephadm-status 
ceph orch ps --format yaml >> $LOG/ceph/cephadm-status 
ceph orch device ls --format yaml >> $LOG/ceph/cephadm-status
ceph orch orch upgrade status >> $LOG/ceph/cephadm-status
```

Reply:
Added (although I called the log file `ceph-orch-status)